src/js/components/MainPage/MainPage.js

```diff
@@ -1,20 +1,104 @@
+import { EXCEPTION_MESSAGE } from '../../constant.js';
 import Component from '../../core/Component.js';
+import { rootStore } from '../../store/rootStore.js';
+import NotFound from '../SearchModal/NotFound.js';
+import SavedVideoCardList from './SavedVideoCardList.js';
+import { savedVideosStorage } from '../../localStorage/savedVideos';
 
 export default class MainPage extends Component {
+  setup() {
+    const savedVideos = savedVideosStorage.load();
+    const watchedVideos = savedVideos.filter(video => video.watched === true);
```

Comment:
```suggestion
    const watchedVideos = savedVideos.filter(video => video.watched);
```
위와 같이 더 단순하게 표현할 수 있어요 😄

Reply:
앗 그렇네요,, 반영했습니다!